docs/methods.rst

```diff
@@ -114,6 +114,39 @@ To continue a run, specify the ``mcwalkerset`` element before your VMC/DMC block
 
 In the project id section, make sure that the series number is different from any existing ones to avoid overwriting them.
 
+.. _transition_guide:
+
+Transition guide for batched drivers
+------------------------------------
+Under the Exascale Computing Project effort, we developed a new set of QMC drivers, called "batched drivers",
+with a unique feature of batching walkers on demand. Available drivers are ``vmc_batch``, ``dmc_batch`` and ``linear_batch``.
```

Comment:
What does "on demand" mean here?  What difference does is make that batching walkers is on demand or not?  (I'm trying to clarify what this means for users)